packages/kbn-securitysolution-grouping/src/components/empty_resuls_panel.tsx

```diff
@@ -1,15 +1,15 @@
 /*
  * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
  * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
```

Comment:
nit: typo in filename

empty_resul**t**s_panel.tsx

Reply:
will fix in the follow up, thanks!